plugins/json-logger/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "my-plugin",
+  "version": "0.0.1",
+  "description": "Description of your plugin",
+  "main": "dist/index.js",
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "dev": "webpack --watch",
+    "build": "webpack",
+    "prebuild": "rimraf dist",
+    "format:write": "prettier --write \"**/*.{js,ts}\""
+  },
+  "author": "Amplication Contributor",
```

Comment:
As you're the auther of the plugin you can provide your name & email adress if desired in the following way: `"Levi van Noort <levi@amplication.com>"`